src/js/module.js

```diff
@@ -211,19 +215,19 @@ iotjs_module_t.normalizePath = function(path) {
 
 
 iotjs_module_t.load = function(id, parent) {
-  if (process.builtin_modules[id]) {
-    iotjs_module_t.curr = id;
-    return Native.require(id);
-  }
-  var module = new iotjs_module_t(id, parent);
-  var modPath = iotjs_module_t.resolveModPath(module.id, module.parent);
-
+  var modPath = iotjs_module_t.resolveModPath(id, parent);
   var cachedModule = iotjs_module_t.cache[modPath];
   if (cachedModule) {
     iotjs_module_t.curr = modPath;
     return cachedModule.exports;
   }
 
+  if (process.builtin_modules[id]) {
+    iotjs_module_t.curr = id;
+    return Native.require(id);
```

Comment:
It seems nobody write the builtin module into cache.

Reply:
yeah, i review my code, it don't seems necessary. when rewrite require.cache with native module id, it returns in the code logic above immediately. I see Native.require have already cache, so pass it, never mind :)